tiered-storage/jcloud/src/main/java/org/apache/bookkeeper/mledger/offload/jcloud/OffloadIndexBlockBuilder.java

```diff
@@ -64,21 +64,19 @@
     OffloadIndexBlockBuilder withDataBlockHeaderLength(long dataHeaderLength);
 
     /**
-     * Finalize the immutable OffloadIndexBlock
+     * Finalize the immutable OffloadIndexBlock.
```

Comment:
Unrelated change

Reply:
Corrected check style violation. - The first sentence of a comment block must end with a period